package.json

```diff
@@ -4,12 +4,10 @@
   "description": "This is a home of Apify documentation.",
   "main": "index.js",
   "scripts": {
-    "build": "npm run lint:md && npm run lint:code && node ./src/build.js",
+    "build": "npm run lint:fix && node ./src/build.js",
     "test": "npm run test",
-    "lint:md": "./node_modules/.bin/markdownlint '**/*.md'",
-    "lint:md:fix": "./node_modules/.bin/markdownlint '**/*.md' --fix",
-    "lint:code": "./node_modules/.bin/eslint . && eslint --ext js,md --fix .",
-    "lint:code:fix": "./node_modules/.bin/eslint . --ext .js,.jsx --fix && eslint --ext js,md --fix ."
+    "lint": "./node_modules/.bin/markdownlint '**/*.md' && ./node_modules/.bin/eslint . && eslint --ext js,md",
+    "lint:fix": "./node_modules/.bin/markdownlint '**/*.md' --fix && ./node_modules/.bin/eslint . --ext .js,.jsx --fix && eslint --ext js,md --fix ."
```

Comment:
Wouldn't be better to have all variants? It makes also package.json more readable:
```
"lint:md": "...",
"lint:md:fix": "...",
"lint:code": "...",
"lint:code:fix": "...",
"lint": "npm run lint:md && npm run lint:code",
"lint:fix": "npm run lint:md:fix && npm run lint:code:fix",
```

Reply:
I think it's more readable as commands are shorter